hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotPurgeRequest.java

```diff
@@ -112,9 +115,16 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
       omClientResponse = new OMSnapshotPurgeResponse(omResponse.build(),
           snapshotDbKeys, updatedSnapInfos,
           updatedPathPreviousAndGlobalSnapshots);
+
+      omMetrics.incNumSnapshotPurges();
+      LOG.info("Successfully executed snapshotPurgeRequest: {{}} along with updating deep clean flags for " +
+              "snapshots: {} and global and previous for snapshots:{}.",
+          snapshotPurgeRequest, updatedSnapInfos.keySet(), updatedPathPreviousAndGlobalSnapshots.keySet());
```

Comment:
Is this intentionally `info` level?

Reply:
Yes. I debated whether to keep it at `info` level or `debug` level. And then decide to keep it at `info` level because whenever we see snapshot chain corruption we don't know how the snapshot info got changed. We don't keep audit logs for the `SnapshotPurge` API because it is an internal API and Ratis logs don't tell all the snapshotInfo objects get updated as part of the purge.
So keeping it at `Info` level for now. Once we have confidence that all the concurrent issues are fixed for the snapshot chain, we can change it to the debug level.

Reply:
Can we make this an audit level log or something? I know audit logs are for users but can we have these logs as some system level audit logs instead of directly writing to om logs?

Reply:
As I said in the previous comment and you also said, audit logs are for user's actions/operations. I don't think it would be a good idea. If you need an actual request, may use the ratis log parser to get the actual request.

Other information that won't be captured in audit logs is the next active snapshot, path previous, and global previous updated in request because audit log will only contain the snapshots that get purged not the snapshots that get updated as part of the purge.

Reply:
Again ideally log level should be debug for this but we need the whole information (all the snapshots get updated as part of the purge) for the snapshot chain corruption.
Once we have confidence that all the concurrent issues are fixed for the snapshot chain, we should change it to the debug level.

Reply:
ok, I'm fine with this